functionality.md

```diff
@@ -19,6 +20,9 @@ end
 
 In order to map the root view of the application to the view of the books controller, you will need to add the following line to the file:
 ```
+# config/routes.rb
+------------------
+
```

Comment:
Random cool markdown trick, if you add the word `ruby` after the three tickmarks, the preview will use a ruby based presenter.

> '''ruby